scene/resources/packed_scene.cpp

```diff
@@ -1743,4 +1744,5 @@ void PackedScene::_bind_methods() {
 PackedScene::PackedScene() {
 
 	state = Ref<SceneState>(memnew(SceneState));
+	mtx = Mutex::create();
```

Comment:
mutex must be deleted manually in the destructor

Reply:
Thanks, I am spoiled rotten by my smart pointers.